src/odb/src/lefin/lefTechLayerSpacingEolParser.cpp

```diff
@@ -362,11 +363,7 @@ void wrongDirSpaceParser(double value,
   sc->setWrongDirSpace(l->dbdist(value));
 }
 
-template <typename Iterator>
-bool parse(Iterator first,
-           Iterator last,
-           odb::dbTechLayer* layer,
-           odb::lefinReader* l)
+bool parse(std::string s, odb::dbTechLayer* layer, odb::lefinReader* l)
```

Comment:
```suggestion
bool parse(const std::string& s, odb::dbTechLayer* layer, odb::lefinReader* l)
```

Reply:
I believe the problem is that many rules are declared
```
qi::rule<std::string::iterator, space_type>
```
and will now need to be
```
qi::rule<std::string::const_iterator, space_type>
```